src/components/DocumentationTopic/PrimaryContent/DeclarationGroup.vue

```diff
@@ -36,6 +61,12 @@ export default {
   name: 'DeclarationGroup',
   components: {
     Source: DeclarationSource,
+    TransitionExpand,
+  },
+  data() {
+    return {
+      selectedIdentifier: this.identifier,
```

Comment:
Do we need the `selectedIdentifier` at all? Why not using the `identifier` directly? Aren't they the same?

Reply:
This is for the purpose of animating the dropdown.

Reply:
To elaborate a bit: to animate the dropdown collapsing without a jump, we visually select the pill users clicked on first and run the animation before we actually push the new route. So there is a split second where the `selectedIdentifier` is different from the identifier of the page